backend/shared/shared/azure_adfs/urls.py

```diff
@@ -1,8 +1,39 @@
 from django.urls import include, path, re_path
+from django_auth_adfs.views import OAuth2LoginView, OAuth2LogoutView
 
 from shared.azure_adfs.views import HelsinkiOAuth2CallbackView
+from shared.common.views import MockEnabledProxyView
+
+from .mock_views import (
+    MockOAuth2CallbackView,
+    MockOAuth2LoginView,
+    MockOAuth2LogoutView,
+)
+
+app_name = "django_auth_adfs"
 
 urlpatterns = [
-    re_path(r"^callback$", HelsinkiOAuth2CallbackView.as_view(), name="callback"),
+    re_path(
+        r"^callback$",
+        MockEnabledProxyView(
+            real_view_class=HelsinkiOAuth2CallbackView,
+            mock_view_class=MockOAuth2CallbackView,
+        ),
+        name="callback",
+    ),
+    re_path(
+        r"^login$",
+        MockEnabledProxyView(
+            real_view_class=OAuth2LoginView, mock_view_class=MockOAuth2LoginView
+        ),
+        name="login",
+    ),
+    re_path(
+        r"^logout$",
+        MockEnabledProxyView(
+            real_view_class=OAuth2LogoutView, mock_view_class=MockOAuth2LogoutView
+        ),
+        name="logout",
+    ),
```

Comment:
Writing tests for these mocked versions would be good for maintainability, like e.g. in backend/shared/shared/oidc/tests/test_mock_views.py but for these endpoints.